docs/source/markdown/podman-manifest-push.1.md

```diff
@@ -32,6 +32,10 @@ environment variable. `export REGISTRY_AUTH_FILE=path`
 Use certificates at *path* (\*.crt, \*.cert, \*.key) to connect to the registry. (Default: /etc/containers/certs.d)
 Please refer to containers-certs.d(5) for details. (This option is not available with the remote Podman client, including Mac and Windows (excluding WSL2) machines)
 
+#### **--compression-format**=**gzip** | *zstd* | *zstd:chunked*
```

Comment:
Is there a reason to not just make this --compression=?
I don't see the -format as adding a lot of value, and I am not a big fan of `-` in the middle of flags.

Reply:
Don't you need the same change in podman-push.1.md also?

Reply:
Actually I have copied this directly from `docs/source/markdown/podman-push.1.md`.

So that also explains the naming using a dash in the middle, I have reused the flag name Giuseppe used when he initially introduced it in `podman image push`.

Reply:
Ok, then I guess we are stuck with it.  I thought this was a new option.